addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -37,8 +37,11 @@ def _get_l10n_it_amount_split_payment(self):
         amount = 0.0
         if self.is_invoice(True):
             for line in [line for line in self.line_ids if line.tax_line_id]:
```

Comment:
```suggestion
            for line in self.line_ids:
```
We can add the `tax_line_id`condition to the `if` statement below :nerd_face:

Reply:
Done.